packages/fastify-vite/config.js

```diff
@@ -80,19 +80,20 @@ const DefaultConfig = {
     return (error, req, reply) => {
       if (config.dev) {
         console.error(error)
+        scope.vite.devServer.ssrFixStacktrace(error);
       }
-      scope.vite.devServer.ssrFixStacktrace(error)
       scope.errorHandler(error, req, reply)
     }
   },
 }
 
 async function configure (options = {}) {
-  const defaultConfig = Object.create(DefaultConfig)
+  const defaultConfig = { ...DefaultConfig }
   const root = resolveRoot(options.root)
-  const [vite, viteConfig] = await resolveViteConfig(root)
+  const dev = typeof options.dev === 'boolean' ? options.dev : defaultConfig.dev
+  const [vite, viteConfig] = await resolveViteConfig(root, dev)
   const clientModule = defaultConfig.clientModule ?? resolveClientModule(vite.root)
-  const bundle = await resolveBundle({ ...options, vite })
+  const bundle = await resolveBundle({ dev, vite })
   const config = Object.assign(defaultConfig, {
```

Comment:
either user set dev to true/false or you get it from `--dev` from the default config